source/agora/common/BanManager.d

```diff
@@ -38,7 +39,7 @@ public class BanManager
         public size_t max_failed_requests = 100;
 
         /// How long does a ban lasts, in seconds (default: 1 day)
-        public Duration ban_duration = 1.days;
+        public @fromSeconds Duration ban_duration = 1.days;
```

Comment:
Where is `@fromSeconds` defined and what does the @ mean?

Reply:
It is defined in `ConfigAttributes.d`.

Reply:
For ref: https://dlang.org/spec/attribute.html#uda